taxi_service/settings.py

```diff
@@ -143,3 +145,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice. This is redundant and should be defined only once to avoid confusion.